pkg/controller/populators/populator-base.go

```diff
@@ -259,7 +259,7 @@ func (r *ReconcilerBase) reconcile(req reconcile.Request, populator populatorCon
 
 	// We first perform the common reconcile steps.
 	// We should only continue if we get a valid PVC'
-	pvcPrime, err := r.reconcileCommon(pvc, populator, log)
```

Comment:
Should we remove the log passed to reconcile too? I don't remember if I added this for some reason or just by mistake.

Reply:
Lets see if we can make it clearer this is the case otherwise I will do this again. I was like why are we passing this log, the struct has a logger.